core/zip/inc/Compression.h

```diff
@@ -86,7 +86,7 @@ enum ECompressionSetting {
 };
 
 /// Deprecated name, do *not* use:
-ROOT_DEPRECATED static constexpr ECompressionAlgorithm kUseGlobalSetting = kUseGlobalCompressionAlgorithm;
+_R__DEPRECATED_LATER("Deprecated name") static constexpr ECompressionAlgorithm kUseGlobalSetting = kUseGlobalCompressionAlgorithm;
```

Comment:
Please use an informational message.

Reply:
Is there a reason you're not using `R__DEPRECATED(6,18,...)` instead of `_R__DEPRECATED_LATER`?